examples/src/components/AutoComplete.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import createClass from 'create-react-class';
+import PropTypes from 'prop-types';
+import Select from 'react-select';
+
+const STATES = require('../data/states');
+
+var AutoCompleteField = createClass({
+	displayName: 'AutoCompleteField',
+	propTypes: {
+		label: PropTypes.string,
+		searchable: PropTypes.bool,
+	},
+	componentDidMount() {
+		// reveal hidden input for testing
+		document.querySelector('[autocomplete="country"]').classList.remove('Select-hidden');
+	},
+	getDefaultProps () {
+		return {
+			label: 'States:',
+			searchable: true,
+		};
+	},
+	getInitialState () {
+		return {
+			selectValue: '',
+			clearable: true,
+			rtl: false,
+		};
+	},
+	switchCountry (e) {
+		var newCountry = e.target.value;
+		this.setState({
+			country: newCountry,
+			selectValue: null,
+		});
+	},
+	updateValue (newValue) {
+		this.setState({
+			selectValue: newValue,
+		});
+	},
+	toggleCheckbox (e) {
+		let newState = {};
+		newState[e.target.name] = e.target.checked;
+		this.setState(newState);
+	},
+	render () {
+		var options = [
+			{ value: 'UK', label: 'United Kingdom' },
+			{ value: 'ES', label: 'Spain' },
+			{ value: 'DE', label: 'Germany' },
```

Comment:
We should probably add a few more countries here, so that users and other contributors can also test this with their specific autofill country.